### DISCUSSION TITLE
Feature request discussion: update apps from RSS/Atom feed

### DISCUSSION THREAD

**Discussion by kenderipa** [ImranR98/Obtainium#867] [Ideas]
Hello,

first of all, thanks for the great app, I really enjoy it and can imagine going away from F-Droid completely because of it. However, there is one aspect of Obtainium I would like to see modified. Obtainium uses API calls for getting Github and Gitlab data, and everything works peachy while an access token is used. However, I would like to delete my Github account in the near future, and herein the problems start. Without access token API requests get blocked in the middle of updating my 100+ android apps that I have on my radar. Would it be possible/plausible to add optional updates via RSS/Atom feed? That way request number is not restricted, and I and a number of other people can enjoy their accountless life with the best way for updating Andoid apps.

Thanks!

**Comment by ImranR98** [comment]
The GitHub RSS feed does not provide APK links: https://github.com/ImranR98/Obtainium/releases.atom

It's also missing a lot of other information like the pre-release label so it would not be compatible with the additional options we have for the github source. We used to scrape the website directly but that stopped working - see #7. So the API is the only option.
